sdk/search/Azure.Search.Documents/src/Generated/Models/SynonymTokenFilter.cs

```diff
@@ -48,7 +48,7 @@ internal SynonymTokenFilter(string oDataType, string name, IList<string> synonym
         }
         /// <summary> A value indicating whether to case-fold input for matching. Default is false. </summary>
         public bool? IgnoreCase { get; set; }
-        /// <summary> A value indicating whether all words in the list of synonyms (if =&gt; notation is not used) will map to one another. If true, all words in the list of synonyms (if =&gt; notation is not used) will map to one another. The following list: incredible, unbelievable, fabulous, amazing is equivalent to: incredible, unbelievable, fabulous, amazing =&gt; incredible, unbelievable, fabulous, amazing. If false, the following list: incredible, unbelievable, fabulous, amazing will be equivalent to: incredible, unbelievable, fabulous, amazing =&gt; incredible. Default is true. </summary>
+        /// <summary> A value indicating whether all words in the list of synonyms (if => notation is not used) will map to one another. If true, all words in the list of synonyms (if => notation is not used) will map to one another. The following list: incredible, unbelievable, fabulous, amazing is equivalent to: incredible, unbelievable, fabulous, amazing => incredible, unbelievable, fabulous, amazing. If false, the following list: incredible, unbelievable, fabulous, amazing will be equivalent to: incredible, unbelievable, fabulous, amazing => incredible. Default is true. </summary>
```

Comment:
The CI is going to complain because the generated code changed, but [it's an easy fix](https://github.com/azure/autorest.csharp#changing-member-doc-comment) if this is our best option.  I do worry that we'll need to be able to escape these characters for our tooling to treat doc comments as XML though.

Btw, it looks like this does work correctly in code samples at least ([source](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core/src/SyncAsyncEventArgs.cs#L40)/[rendered docs](https://docs.microsoft.com/en-us/dotnet/api/azure.syncasynceventargs.isrunningsynchronously?view=azure-dotnet#remarks)).

Reply:
I think the easiest path to resolution, at least docs wise, is to PR the above wherever we use it.

I have confirmed our docs still render this properly:
![image](https://user-images.githubusercontent.com/45376673/132402540-5e8f7c6a-7cc5-4f2b-b48f-2daa3c036dca.png)

Going to run a second validation to confirm that our docs CI also sees the same thing.

Reply:
That being said, I'm still ok with filing that issue with docs to get them to render html escape characters properly. I think you're probably right to be worried about not being able to escape characters in the future.

Reply:
Is there a need to fix one instance of this issue manually? For the same class, the [`Synonyms` property doc](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.search.models.synonymtokenfilter?view=azure-dotnet) also has this issue.
The ctor param docs are generated correctly though.

Reply:
Not to mention we use this escaped character formatting in hundreds of places across the .NET SDK.

Reply:
Hmmm. But what about stuff like [this](https://github.com/Azure/azure-sdk-for-net/blob/6ea1f971788872fdaf0718a6ee7d3d6429ead90d/sdk/search/Azure.Search.Documents/src/SearchClient.cs#L513) where we really _do_ need those to render properly without calling it an html tag. 

I'll submit a PR that fixes the descriptions, but definitely going to be submitting a request with the docs team.

Reply:
Another pertinent point - the issue is with the [docs in the track1 SDK](https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.search.models.synonymtokenfilter?view=azure-dotnet). The [track 2 SDK docs](https://docs.microsoft.com/en-us/dotnet/api/azure.search.documents.indexes.models.synonymtokenfilter?view=azure-dotnet) are rendered properly.

Reply:
So, the file being updated in this PR is not the issue.

Reply:
Wait. What? I was looking at track 2 on docs.ms I thought. 😬

Reply:
The complaint is about Microsoft.Azure* class. I don't think the Service team also cares about it that much. :)
I guess they stumbled upon this type with the same name when looking for track2 stuff.

Reply:
Yeah. The PR is actually for the wrong document. That's what was confusing me. 

I'm going to figure out why the track 1 is rendering like that before I call this fine. This PR won't live though.

Reply:
Ok. Ran down the issue. I'll respond on the thread. Thanks.